src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -113,6 +116,7 @@ const initialState = {
   isCCEligible: false,
   hideUpdateAddressAlert: false,
   requestLocationStatus: FETCH_STATUS.notStarted,
+  communityCareProviderList: [],
```

Comment:
You should also add `requestStatus: FETCH_STATUS.notStarted` here, too.

Reply:
Added.